### DISCUSSION TITLE
在线demo的高精度版本是最近新增的吧，对应哪个模型呢？

### DISCUSSION THREAD

**Discussion by shinerdeng** [PaddlePaddle/PaddleOCR#13974] [Q&A]
![image](https://github.com/user-attachments/assets/70ae6ca6-1f5e-4cb7-a79e-ad30bab2fedc)
这个地址https://aistudio.baidu.com/community/app/91660/webUI
这个高精度模型是新增的吧，开源了吗？比我本地v4版本100多MB那个效果要好

**Comment by GreatV** [comment]
https://paddlepaddle.github.io/PaddleOCR/latest/model/index.html
